src/harness/BenchmarkDotNet.Extensions/PerfLabExporter.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Parameters;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using Reporting;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace BenchmarkDotNet.Extensions
+{
+    internal class PerfLabExporter : ExporterBase
+    {
+        protected override string FileExtension => "json";
+        protected override string FileCaption => "perf-lab-report";
+        public PerfLabExporter()
+        {
+        }
+        public override void ExportToLog(Summary summary, ILogger logger)
+        {
+            var reporter = Reporter.CreateReporter();
+            if (reporter == null) // not running in the perf lab
+                return;
+
+            var nameDelegate = typeof(BenchmarkCase).Assembly.GetType("BenchmarkDotNet.Exporters.FullNameProvider").GetMethod("GetBenchmarkName", BindingFlags.Static | BindingFlags.NonPublic);
+
+
+            foreach (var report in summary.Reports)
+            {
+                var test = new Test();
+                //test.Name = FullNameProvider.GetBenchmarkName(report.BenchmarkCase);
+                test.Name = (string)nameDelegate.Invoke(null, new[] { report.BenchmarkCase });
+                test.Categories = report.BenchmarkCase.Descriptor.Categories;
+                var results = from result in report.AllMeasurements
+                              where result.IterationMode == Engines.IterationMode.Workload && result.IterationStage == Engines.IterationStage.Result
+                              orderby result.LaunchIndex, result.IterationIndex
+                              select new { result.Nanoseconds, result.Operations };
+                test.Counters.Add(new Counter
+                {
+                    Name = "Duration of single invocation",
+                    TopCounter = true,
+                    DefaultCounter = true,
+                    HigherIsBetter = false,
+                    MetricName = "ns",
+                    Results = (from result in results
+                               select result.Nanoseconds / result.Operations).ToList()
+                });
+                test.Counters.Add(new Counter
+                {
+                    Name = "Duration",
```

Comment:
We used to have this metric in BenchView, when we were using the `IterationTime` as the result because xunit-performance did not scale the results (`Scaled=IterationTime/InnerInvocationTime`) But BDN scales the results and reports the mentioned above "Duration of a single invocation" which is the scaled value.

More: https://github.com/dotnet/performance/blob/master/docs/porting-to-benchmarkdotnet.md#scaling-of-the-results

With BDN we can easily change the `IterationTime` and it's going to use the provided value and scale the results:

https://github.com/dotnet/performance/blob/07b782ef1efb2acad301c6ec74f24c2e30ce802b/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs#L18

For example: in the future, we could change it to 200ms to save 1/5 of the execution time (if we make sure it does not affect the results). If we ever do that, the "Duration" will change but "Duration of single invocation" will not. 

We should consider supporting only "Duration of single invocation" in the new reporting system. Or even choose a better name ;)

Reply:
Thanks for the info! I chose to report everything that was being reported for continuity. "Duration of a single invocation" is the "default" and will be what people see first. Other counters like this are for additional context. We can always turn one off if it turns out to not be worth it.